research/blueprint_algo/average_strategy_short_deck.py

```diff
@@ -53,6 +58,7 @@ def average_strategy(all_file_paths: List[str]) -> Dict[str, Dict[str, float]]:
         # Sum probabilities from computed strategy..
         for info_set, this_info_sets_regret in sorted(regret.items()):
```

Comment:
are there any cases you have seen where `this_info_sets_regret` is an empty dict? I'm suprised there are!

Reply:
Yes! So, especially during cfr during the non-traversing player's turn, we lookup the infoset in the agent's regret. However Agent.regret is a defaultdict which defaults to {}. We haven't updated regret, but we are now averaging the strategy with a default strategy.

Reply:
Best case scenario is to only include strategies that were learned.